sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -316,26 +319,27 @@ export class SecretClient {
     const requestOptions = operationOptionsToRequestOptionsBase(options);
 
     if (requestOptions) {
-      const { enabled, notBefore, expiresOn: expires, ...remainingOptions } = requestOptions;
-      const unflattenedOptions = {
-        ...remainingOptions,
+      const { enabled, notBefore, expiresOn: expires } = requestOptions;
+
+      const span = this.createSpan("updateSecretProperties", requestOptions);
+      const optionalParams: SecretUpdateParameters = {
+        contentType: options.contentType,
+        tags: options.tags,
         secretAttributes: {
           enabled,
           notBefore,
           expires
         }
```

Comment:
I find it interesting how different this is from: https://github.com/Azure/azure-sdk-for-js/pull/9727/files#diff-b7f79eb11bce1239d2cc0fea068e4086R606